datahub-web-react/src/app/permissions/policy/ManagePolicies.tsx

```diff
@@ -258,17 +262,28 @@ export const ManagePolicies = () => {
             title: `Delete ${policy?.name}`,
             content: `Are you sure you want to remove policy?`,
             onOk() {
-                deletePolicy({ variables: { urn: policy?.urn as string } }); // There must be a focus policy urn.
-                analytics.event({
-                    type: EventType.DeleteEntityEvent,
-                    entityUrn: policy?.urn,
-                    entityType: EntityType.DatahubPolicy,
-                });
-                message.success('Successfully removed policy.');
-                setTimeout(() => {
-                    policiesRefetch();
-                }, 3000);
-                onCancelViewPolicy();
+                removeFromListPoliciesCache(client, policy?.urn , page, pageSize, query);
```

Comment:
this is all we should need

Reply:
in other places we have same no. of parameter         removeFromListIngestionSourcesCache(client, urn, page, pageSize, query);